scripts/create_image.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+mkdir $1
```

Comment:
sanity check for the number of arguments and show a usage message if it's not the case